fe1-web/src/features/evoting/network/messages/__tests__/SetupElection.test.ts

```diff
@@ -24,6 +25,10 @@ const TIMESTAMP_BEFORE = new Timestamp(1609445600);
 let electionId: Hash;
 let mockQuestionObject1: Question;
 let mockQuestionObject2: Question;
+
+// As discussed on slack, in these tests we should assume that the input to the messages is
```

Comment:
I'm sure the Slack discussion will be quickly forgotten or unknown to most readers 😄 
```suggestion
// In these tests, we should assume that the input to the messages is
```

Reply:
Good point 👍